temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` for debugging in committed code. It is better to remove it or use logging for debugging purposes.
```
import logging

# Set up basic configuration for logging
logging.basicConfig(level=logging.DEBUG)

# Example usage
logging.debug('Debugging message')
```